### ISSUE TITLE
Initializing shadcn/ui can be slow or stuck for a long time

### ISSUE THREAD

**Issue by mhaidarhanif** [Code-Forge-Net/Remix-Forge#4]
Hi Alem,

First of all, thank you for making Remix Forge.

Currently I'm trying to initialize shadcn/ui on the fresh Remix install (v1.17.1) but it just stuck like this.

<img width="791" alt="image" src="https://github.com/Code-Forge-Net/Remix-Forge/assets/235584/1d53a61f-1e48-414a-82be-2558d7a2289b">

There's an error from Output > Extension Host like this:

<img width="1279" alt="image" src="https://github.com/Code-Forge-Net/Remix-Forge/assets/235584/b2e96e7c-71f5-49be-b366-6d5cb3af9189">

Thank you.

**Comment by AlemTuzlak** [comment]
Thank you @mhaidarhanif for the kind words and the submission, the output is really helpful, I will fix this asap because I already think I knowwhat the issue is!

**Comment by mhaidarhanif** [comment]
You're welcome @AlemTuzlak 

Awesome! Looking forward to it.

**Comment by mhaidarhanif** [comment]
In the meantime I'm following the docs in shadcn UI: https://ui.shadcn.com/docs/installation

**Comment by AlemTuzlak** [comment]
@mhaidarhanif published a new release, should be up in 5-10m, let me know if this fixes the issue for you, it should because I think I found the culprit

**Comment by mhaidarhanif** [comment]
@AlemTuzlak Thank you, it works now! Just had the time to try it again recently.

It's also nice to use the latest shadcn UI theming workflow.

Although there are some things we'll have to to fix manually:

<img width="1222" alt="image" src="https://github.com/Code-Forge-Net/Remix-Forge/assets/235584/e3ce53a5-ad4b-4dae-9ac5-4c2a63aba7a2">

This is from the fresh latest install of Remix.

**Comment by AlemTuzlak** [comment]
@mhaidarhanif yeah after a discussion on discord yesterday I think I will need to mark this as experimental for now, I had to sit down and rewrite almost the whole code after shad changed the CLI and I have a feeling from his tweets it will happen again so until his CLI is stable I don't think this can be stable either

**Comment by AlemTuzlak** [comment]
I will be updating the code to remove the tailwind stuff, for the folders I need to dee if there is a way for me to ask for permission to delete a folder when an extension doesn't have the right

**Comment by mhaidarhanif** [comment]
@AlemTuzlak Yes, I think for now it's okay to just let people know that this particular part is experimental and we might have to fix just a little bit.

Also several days later shadcn will release the update for the theming again.

**Comment by mhaidarhanif** [comment]
Looks like soon we had to be aware of the registry: https://ui.shadcn.com/docs/components/calendar

<img width="543" alt="image" src="https://github.com/Code-Forge-Net/Remix-Forge/assets/235584/e3bd4ac3-c80a-4bf2-98f2-6ec856d5584b">

**Comment by AlemTuzlak** [comment]
Interesting, I definitely will create a new version and update the fact accordingly that this is experimental until the final stable version lands then I will go through all of it again and make it work for Remix

**Comment by mhaidarhanif** [comment]
Nice, it's already good by now :)

All workflows are working fine for this UI generator.

As shadcn UI also supports Remix from the website as well.
